services/py-api/src/server/logger/logger_factory.py

```diff
@@ -83,9 +82,8 @@ def configure_app_logger(env: str) -> None:
             add_log_level,
             merge_contextvars,
             TimeStamper(fmt="%Y-%m-%d %H:%M:%S"),
-            ExceptionPrettyPrinter(),
```

Comment:
This is removed as we have the `format_exc_info`, right? At least this is what the docs say " This processor is mostly for development and testing so you can read exceptions properly formatted.  It behaves like `format_exc_info` except it removes the exception data from the event dictionary after printing it." Or the StackInfoRender takes care of all of this and the exceptions are properly printed in the server.log and the stderr (in the console)

How are the exceptions printed now to the stdout when ENV=Local and how are they printed to the server.log when ENV=DEV? I think in the `server.log` they would be on one line after the `exception` key, which would make it difficult to read. Again I could be wrong, so pls provide me with screenshots as I cannot test it right now.